src/commands/dev/lib/dev-router.ts

```diff
@@ -53,20 +58,20 @@ export default async function(
         continue;
       }
 
-      if (!src.startsWith('^')) {
-        src = `^${src}`;
-      }
-
-      if (!src.endsWith('$')) {
-        src = `${src}$`;
-      }
+      src = src.replace(/^\^?\/?(.*)$/, (_, path) => {
+        let p = path;
+        if (!p.endsWith('$')) {
+          p += '$';
+        }
+        return `^${p}`;
```

Comment:
```suggestion
        return '^ + p;
```

If the linter will let you, I think this will be more readable

Reply:
The linter does indeed error:

```
  66:16  error    Unexpected string concatenation            prefer-template
```